app/workers/stale_issue_marker.rb

```diff
@@ -5,22 +5,14 @@ class StaleIssueMarker
   include SidekiqWorkerMixin
 
   # If an issue/pr has any of these labels, it will not be marked as stale or closed
-  PINNED_LABELS       = ['pinned', GithubService::Issue::REQUIRES_TRIAGE].freeze
-  SEARCH_SORTING      = {:sort => :updated, :direction => :asc}.freeze
-  STALE_ISSUE_MESSAGE = <<-EOS.freeze
-This issue has been automatically marked as stale because it has not been updated for at least 3 months.
+  PINNED_LABELS  = ['pinned', GithubService::Issue::REQUIRES_TRIAGE].freeze
+  SEARCH_SORTING = {:sort => :updated, :direction => :asc}.freeze
+  COMMENT_FOOTER = <<~FOOTER.sub("ManageIQ\n", "ManageIQ ").strip!
```

Comment:
I don't think this is needed ... Why not just leave the multi-line comment?

Reply:
Cuz GithubFlavoredMarkdown...

If you leave it as multi line:

```
Thank you for all your contributions!  More information about the ManageIQ
triage process can be found in [the traige process documentation][1].

[1]: https://www.manageiq.org/docs/guides/triage_process
```

**It will format like this:**

Thank you for all your contributions!  More information about the ManageIQ
triage process can be found in [the traige process documentation][1].



**Instead of like this:**

Thank you for all your contributions!  More information about the ManageIQ triage process can be found in [the traige process documentation][1].


(make note of the line break after "ManageIQ")


[1]: https://www.manageiq.org/docs/guides/triage_process